src/entities/data-export/data-export.ts

```diff
@@ -77,11 +77,20 @@ export class DataExport extends UniverseEntity<DataExportPayload, DataExportPayl
     }
   }
 
-  public async init (): Promise<DataExport | undefined> {
+  public async fetchData (): Promise<DataExport | undefined> {
     try {
-      await this.post()
-
-      return this
+      const opts = {
+        method: 'POST',
+        url: `${this.apiCarrier?.injectables?.base}/${this.endpoint}`,
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8'
+        },
+        data: this._rawPayload ?? undefined,
+        responseType: 'json'
+      }
+
+      const response = await this.http?.getClient()(opts)
+      return response.data.data
```

Comment:
Question: Usually, the fetch is handled by the base class (which implements all generic crud operations) - I guess this is special because it is a POST request to fetch export data?

Reply:
Yes, and because this doesnt really fit with the entity definition of an item, so it needs different handling, otherwise i would use the generic post.